packages/subgraph/build/schema.graphql

```diff
@@ -0,0 +1,17 @@
+type Greeting @entity {
```

Comment:
`build` and `generated` folders are usually ignored. Any reason to commit it here?

Reply:
also cleaned up here: https://github.com/scaffold-eth/scaffold-eth-2/pull/722